### DISCUSSION TITLE
White mode or customize theme

### DISCUSSION THREAD

**Discussion by chenyao0910** [dailydotdev/daily#1087] [Feature Requests]
I think a feature that can customize the color on every component is cool, if this is too complicated, I think white mode is a essential thing too.

**Comment by nimrodkra** [comment]
Hi, thanks for the feedback! We do support light theme as well, here’s how
to set it up:
https://docs.daily.dev/docs/customize-your-feed/theme#light-theme

We will consider adding more themes and customization options in the future.
